docs/proto/docs.md

```diff
@@ -0,0 +1,398 @@
+    - [JinaDiscoverEndpointsRPC](#jina-JinaDiscoverEndpointsRPC)
+    - [JinaGatewayDryRunRPC](#jina-JinaGatewayDryRunRPC)
+    - [JinaInfoRPC](#jina-JinaInfoRPC)
+    - [JinaRPC](#jina-JinaRPC)
+    - [JinaSingleDataRequestRPC](#jina-JinaSingleDataRequestRPC)
+  
+- [Scalar Value Types](#scalar-value-types)
+
+
+
+<a name="docarray-proto"></a>
+<p align="right"><a href="#top">Top</a></p>
+
+## docarray.proto
+
+
+
+<a name="docarray-DocumentArrayProto"></a>
+
+### DocumentArrayProto
+this file is just a placeholder for the DA coming from docarray dependency
+
+
+
+
+
+ 
+
+ 
+
+ 
+
+ 
+
+
+
+<a name="jina-proto"></a>
+<p align="right"><a href="#top">Top</a></p>
+
+## jina.proto
+
+
+
+<a name="jina-DataRequestListProto"></a>
+
+### DataRequestListProto
+Represents a list of data requests
+This should be replaced by streaming
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| requests | [DataRequestProto](#jina-DataRequestProto) | repeated | requests in this list |
+
+
+
+
+
+
+<a name="jina-DataRequestProto"></a>
+
+### DataRequestProto
+Represents a DataRequest
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| header | [HeaderProto](#jina-HeaderProto) |  | header contains meta info defined by the user |
+| parameters | [google.protobuf.Struct](#google-protobuf-Struct) |  | extra kwargs that will be used in executor |
+| routes | [RouteProto](#jina-RouteProto) | repeated | status info on every routes |
+| data | [DataRequestProto.DataContentProto](#jina-DataRequestProto-DataContentProto) |  | container for docs and groundtruths |
+
+
+
+
+
+
+<a name="jina-DataRequestProto-DataContentProto"></a>
+
+### DataRequestProto.DataContentProto
+
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| docs | [docarray.DocumentArrayProto](#docarray-DocumentArrayProto) |  | the docs in this request |
+| docs_bytes | [bytes](#bytes) |  | the docs in this request as bytes |
+
+
+
+
+
+
+<a name="jina-DataRequestProtoWoData"></a>
+
+### DataRequestProtoWoData
+
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| header | [HeaderProto](#jina-HeaderProto) |  | header contains meta info defined by the user |
+| parameters | [google.protobuf.Struct](#google-protobuf-Struct) |  | extra kwargs that will be used in executor |
+| routes | [RouteProto](#jina-RouteProto) | repeated | status info on every routes |
+
+
+
+
+
+
+<a name="jina-EndpointsProto"></a>
+
+### EndpointsProto
+Represents the set of Endpoints exposed by an Executor
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| endpoints | [string](#string) | repeated | list of endpoints exposed by an Executor |
+
+
+
+
+
+
+<a name="jina-HeaderProto"></a>
+
+### HeaderProto
+Represents a Header.
+- The header&#39;s content will be defined by the user request.
+- It will be copied to the envelope.header
```

Comment:
```suggestion
- It is copied to the envelope.header
```

Use present-simple tense instead of future tense where possible

Reply:
This whole file is generated maybe from my local docs build script. This file should not be included in the pr.

Reply:
Can you remove the file?